cfgov/jinja2/v1/_includes/molecules/pagination.html

```diff
@@ -1,63 +1,79 @@
 
 {# ==========================================================================
 
-   Pagination
+   pagination.render()
 
    ==========================================================================
 
    Description:
 
    Builds pagination markup when given:
 
-   posts:                        TODO: fill in data type and description.
+   total_pages: Total number of pages.
 
-   posts.paginator:              TODO: fill in data type and description.
+   curr_page:   Currently selected page (out of the total).
```

Comment:
`current_page` work? Updated in https://github.com/cfpb/cfgov-refresh/pull/2267/commits/1baeacdfa231d81bcbb8cf5882b0d715296bb93f